### PR TITLE
SHANK-419 Search by state

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vasdvp.atlassian.net/browse/SHANK-419

### CODE REVIEW

**Comment by jblefkove-va** [comment]
hold off

File: facilities/pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <commons-csv.version>1.8</commons-csv.version>
     <docker.imageName>lighthouse-facilities</docker.imageName>
     <guava.version>28.2-jre</guava.version>
-    <jacoco.coverage>0.93</jacoco.coverage>
+    <jacoco.coverage>0.95</jacoco.coverage>
```
[the remark this answers is not in the document]
Comment (jhulbert-va) [reply facilities/pom.xml]: Can't we just remove that now though?

[the remark this answers is not in the document]
Comment (jblefkove-va) [reply facilities/pom.xml]: Good point

**Comment by jblefkove-va** [comment]
Good to go

**Pull request merged**